src/test/regress/spec/isolation_mx_common.include.spec

```diff
@@ -7,6 +7,14 @@ setup
         LANGUAGE C STRICT VOLATILE
         AS 'citus', $$start_session_level_connection_to_node$$;
 
+    CREATE OR REPLACE FUNCTION override_backend_data_command_originator(bool)
+        RETURNS void
+        LANGUAGE C STRICT IMMUTABLE
+        AS 'citus', $$override_backend_data_command_originator$$;
+
+    SELECT run_command_on_workers($$ CREATE OR REPLACE FUNCTION override_backend_data_command_originator(bool)
+        RETURNS void LANGUAGE C STRICT IMMUTABLE AS 'citus';$$);
```

Comment:
Does this not need the `$$override_backend_data_command_originator$$` part?

Reply:
not necessarily, by default PG searches for the same name already.

Having `$$` inside `run_command_on_workers` giving weird syntax errors, hence skipped that (also didn't seem important as this is test-only function)